tests/manage/monitoring/pagerduty/test_deployment_status.py

```diff
@@ -33,17 +34,20 @@ def test_ceph_manager_stopped_pd(measure_stop_ceph_mgr):
 
     # get incidents from time when manager deployment was scaled down
     incidents = measure_stop_ceph_mgr.get("pagerduty_incidents")
-    target_label = constants.ALERT_MGRISABSENT
+    for target_label in [
+        constants.ALERT_MGRISABSENT,
+        constants.ALERT_MGRISMISSINGREPLICAS,
+    ]:
 
-    # TODO(fbalak): check the whole string in summary and incident alerts
-    assert pagerduty.check_incident_list(
-        summary=target_label,
-        incidents=incidents,
-        urgency="high",
-    )
-    api.check_incident_cleared(
-        summary=target_label, measure_end_time=measure_stop_ceph_mgr.get("stop")
-    )
+        # TODO(fbalak): check the whole string in summary and incident alerts
+        assert pagerduty.check_incident_list(
+            summary=target_label,
+            incidents=incidents,
+            urgency="high",
+        )
```

Comment:
When this fails, would you learn which alert is causing the problem? Also, would it be useful to know when both alerts are missing?

Reply:
This can be read from logs. In logs are listed all alerts that were triggered.

Reply:
Ah, ok. You would be the one looking into the logs after all :)